lib/webpack.js

```diff
@@ -15,7 +16,9 @@ function webpack(options, callback) {
 			return webpack(options);
 		}));
 	} else if(typeof options === "object") {
-		if(!options.entry && !options.plugins) {
+		var webpackOptionsValidationErrors = validateWebpackOptions(options);
+		if(webpackOptionsValidationErrors.length) {
+			console.log("'options' validation errors:", webpackOptionsValidationErrors);
```

Comment:
What about users that use webpack programmatically? e.g. most of the React applications will have a setup using `webpack` and `webpack-dev-middleware`.

Do I need to document presence of the `validationErrors` property?